docker/launch-netbox.sh

```diff
@@ -22,27 +47,37 @@ load_configuration() {
   # this curl call will get a reply once unit is fully launched
   curl --silent --output /dev/null --request GET --unix-socket $UNIT_SOCKET http://localhost/
 
-  echo "⚙️ Applying configuration from $UNIT_CONFIG"
+  if [[ -n "$UNIT_CONFIG" ]] && [[ -s "$UNIT_CONFIG" ]]; then
+    echo "⚠️ The UNIT_CONFIG environment variable is deprecated. All *.pem and *.json files in /etc/unit will be loaded automatically when UNIT_CONFIG is undefined."
+    echo "⚙️ Applying configuration from UNIT_CONFIG environment variable: $UNIT_CONFIG"
+    put_config $UNIT_CONFIG "config" || return 1
+  else
+    echo "🔍 Looking for certificate bundles in /etc/unit/..."
```

Comment:
I dislike that the folder is now hardcoded to `/etc/unit`.

```suggestion
    UNIT_CONFIG_DIR=${UNIT_CONFIG_DIR-/etc/unit/}
    echo "🔍 Looking for certificate bundles in '${UNIT_CONFIG_DIR}'"
```

Instead of `/etc/unit/` you'd just use the `UNIT_CONFIG_DIR` variable instead.